tests/unit/test_dashboards.py

```diff
@@ -751,6 +806,31 @@ def test_dashboards_creates_dashboard_with_widget_below_text_widget(tmp_path):
     ws.assert_not_called()
 
 
+def test_dashboards_creates_dashboard_with_id_collisions(tmp_path):
```

Comment:
@nfx : What do we want from a user perspective in this case?

Examples:
- Not allow any id collisions
- Not allow id collisions when specified in file header
- Not allow specifying id in the dashboard yaml (maybe does not make sense there any way)
- Allow id collisions and metadata in the dashboard yaml apply to all matches

Reply:
> Not allow specifying id in the dashboard yaml (maybe does not make sense there any way)

I think we should do this as it will be confusing to resolve the following metadata, also without collisions:

``` yaml
tiles:
  a:
    id: b
  b:
    id: z
  c:
    id: a
  d:
    id: f
  e:
    width: 10
  f:
    id: d
```

Reply:
Created issue: https://github.com/databrickslabs/lsql/issues/199